data/json/items/gunmod/accessories.json

```diff
@@ -132,5 +132,33 @@
     "location": "belt clip",
     "mod_targets": [ "rugerlcp", "kp32", "kp3at", "kpf9", "cop_38", "moss_brownie" ],
     "flags": [ "IS_ARMOR", "SKINTIGHT", "WATER_FRIENDLY" ]
+  },
+  {
+    "id": "12_gauge_shell_holder",
+    "type": "GUNMOD",
+    "name": { "str": "12 gauge shell holder" },
+    "description": "Designed to be affixed to the side of a shotgun's receiver, this polymer shotshell holder can store six 12 gauge shells for easy reloading.",
+    "weight": "16 g",
+    "volume": "60 ml",
+    "price": 2000,
+    "price_postapoc": 50,
+    "install_time": "30 s",
+    "material": [ "plastic" ],
+    "flags": [ "NOT_MAGAZINE", "TARDIS" ],
+    "symbol": ":",
+    "color": "black",
+    "location": "loading port",
```

Comment:
```suggestion
    "location": "rail",
```
Stock-mounted versions also exist https://www.midwayusa.com/product/1014560565?pid=668092
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/99621099/2a570d08-99bf-44bc-b68c-4f813f71476f)

Reply:
Isn't the rail on the top of the gun? It wouldn't go up there.

Stock mounted versions are planned. This is a draft.

Reply:
The "rail" rail is on the side. It's where non-underbarrel laser sights go. (There are also underbarrel and top rails, which are referred to in-game as "underbarrel" and "sights" locations.)

On pump-action (and modern automatic) shotguns, the side rail would likely be on the forend (the thing you "pump" after each shot, the thing that goes *chuck-chuck*). This isn't an adequate position for a shell holder, since a filled holder would interfere with one's ability to grip the forend. See this replacement forend with an underbarrel rail and two side rails:

https://trinitysupply.com/cdn/shop/files/2_042cc13a-522b-42f6-9c74-c1022d30baed.jpg

But since we don't have a "body" location or something, rails would do.

Reply:
I think I'll just put them on the stock then, if the side rail is a kludgy fix.

Reply:
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/94415528/a7c4f531-19b7-4f2e-936f-f5b222e8c82f)

I don't see any that fit on the forend, but I see a bunch that straddle the receiver and add a top rail to the gun.

Reply:
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/94415528/2fe4282a-2916-4f5d-b08e-7b5645235dc4)
I guess there are these, but this doesn't look like something that would fit on any old gun. I think my PR will just add generally-applicable mods and I'll leave the hyper-specifics to others.

Reply:
They don't necessarily go over the top. Looks like they're mostly held by the pins(?) on the shotgun's body, as can be seen in the (poor-quality, but legible) instructions on this product:

https://www.amazon.com/TacStar-6-Shot-Sidesaddle-Black/dp/B001AMZXQI

Which, again, might require a special slot on shotguns, which may or may not be beyond this PR's scope.